src/main/java/org/jitsi/jicofo/event/BridgeEvent.java

```diff
@@ -139,6 +178,17 @@ public String getBridgeJid()
     }
 
     /**
+     * Obtains the value of video stream count associated with the current
+     * <tt>BridgeEvent</tt>.
+     * @return <tt>Integer</tt> or <tt>null</tt> if attribute not present for
```

Comment:
The property is always set, so we don't have to worry about it, but wouldn't this throw a `ClassCastException`, if the property is not present?